migrations/versions/503d27d0a21a_update_dino_cube_and_fmc_events_.py

```diff
@@ -0,0 +1,29 @@
+"""Update Dino Cube and FMC events descriptions
+
+Revision ID: 503d27d0a21a
+Revises: 555566667777
+Create Date: 2020-10-09 13:23:54.090903
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '503d27d0a21a'
+down_revision = '555566667777'
+branch_labels = None
+depends_on = None
+
+
+def upgrade():
+    events = table('events', column('name', String), column('description', String))
+    op.execute(events.update().where(events.c.name == op.inline_literal('Dino Cube')).values({'description': op.inline_literal('<p>R indicates a clockwise turn of the UFR corner, and L indicates a clockwise turn of the UFL corner. Any solved state is considered solved</p>')}))
```

Comment:
Thanks for adding the blurb about any solved state being solved! Nitpick, can you add a period at the end?

Reply:
Sure thing!